gravitee-repository-elasticsearch/src/main/java/io/gravitee/repository/elasticsearch/analytics/query/DateHistogramQueryCommand.java

```diff
@@ -47,7 +47,14 @@
 	@Override
 	public DateHistogramResponse executeQuery(Query<DateHistogramResponse> query) throws AnalyticsException {
 		final DateHistogramQuery dateHistogramQuery = (DateHistogramQuery) query;
-		final String sQuery = this.createQuery(TEMPLATE, query);
+s
+		final Long from = dateHistogramQuery.timeRange().range().from();
+		final Long to = dateHistogramQuery.timeRange().range().to();
+
```

Comment:
Add a small comment to explain the benefits of this, by using internal ES cache...